nextcloud/content.md

```diff
@@ -499,99 +545,65 @@ The `--pull` option tells docker to look for new versions of the base image. The
 You're already using Nextcloud and want to switch to docker? Great! Here are some things to look out for:
 
 1.	Define your whole Nextcloud infrastructure in a `docker-compose` file and run it with `docker-compose up -d` to get the base installation, volumes and database. Work from there.
-
 2.	Restore your database from a mysqldump (nextcloud\_db\_1 is the name of your db container)
-
-	-	To import from a MySQL dump use the following commands
-
-	```console
-	docker cp ./database.dmp nextcloud_db_1:/dmp
-	docker-compose exec db sh -c "mysql --user USER --password PASSWORD nextcloud < /dmp"
-	docker-compose exec db rm /dmp
-	```
-
-	-	To import from a PostgreSQL dump use to following commands
-
-	```console
-	docker cp ./database.dmp nextcloud_db_1:/dmp
-	docker-compose exec db sh -c "psql -U USER --set ON_ERROR_STOP=on nextcloud < /dmp"
-	docker-compose exec db rm /dmp
-	```
-
+	-	To import from a MySQL dump use the following commands`console
+		docker cp ./database.dmp nextcloud_db_1:/dmp
+		docker-compose exec db sh -c "mysql --user USER --password PASSWORD nextcloud < /dmp"
+		docker-compose exec db rm /dmp
+		`
+	-	To import from a PostgreSQL dump use to following commands`console
+		docker cp ./database.dmp nextcloud_db_1:/dmp
+		docker-compose exec db sh -c "psql -U USER --set ON_ERROR_STOP=on nextcloud < /dmp"
+		docker-compose exec db rm /dmp
+		`
 3.	Edit your config.php
-
 	1.	Set database connection
-
-		-	In case of MySQL database
-
-		```php
-		'dbhost' => 'db:3306',
-		```
-
-		-	In case of PostgreSQL database
-
-		```php
-		'dbhost' => 'db:5432',
-		```
-
-	2.	Make sure you have no configuration for the `apps_paths`. Delete lines like these
-
-		```php
+		-	In case of MySQL database`php
```

Comment:
Again, something seems to have mangled the formatting of these code blocks :sob: